### PR TITLE
Add did:bid:method

### PR DESCRIPTION
**Pull request by teleinfo-bif**
BID provides distributed identifiers and blockchain digital identity services for people, enterprises, devices and digital objects. It aims to build a decentralized, data-secure, privacy-protected and flexible identifier system that addresses trusted connections to people, enterprises, devices and digital objects，enabling the vision of the Internet of Everything and trust ingress with everything.

We create this PR to add [BIF DID protocol Specification](https://github.com/teleinfo-bif/bid/tree/master/doc/en)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -947,7 +947,24 @@ <h1>The Registry</h1>
...
+          PROVISIONAL
+        </td>
+        <td>
+          bif
+        </td>
+        <td>
+          teleinfo caict
+        </td>
+        <td>
+          <a href="https://github.com/teleinfo-bif/bid/blob/master/doc/en/BIF%20DID%20protocol%20Specification_up.md">BIF DID Method</a>
```
Comment (msporny) [inline index.html]: This document doesn't exist, we can't merge w/o a spec.

Comment (msporny) [reply index.html]:
```suggestion
          <a href="https://github.com/teleinfo-bif/bid/tree/master/doc/en">BIF DID Method</a>
```

**Comment by talltree** [comment]
I am confused by this DID method spec. 

- It says it uses the BIF blockchain but there is no reference given for any information about that blockchain.

- The two links given in "Status of this Document" both give 404s.

- It says that you can use nicknames in a client, but it has no other information about how nicknames are assigned or whether they are required to be unique.

- It says you can add personal data to a BID DDO (a term that the DID spec has deprecated) without providing precautions about the use of personal data in DID documents.

In short, this spec is not yet of sufficient quality to qualify for inclusion in the Registry. All of these issues need to be addressed first.
